templates/common/infra/bicep/core/search/search-services.bicep

```diff
@@ -1,3 +1,4 @@
+metadata description = 'Creates a Search Service.'
```

Comment:
@jongio  - not sure about this. Should this be "'Creates an Azure Cognitive Search instance."?